index.html

```diff
@@ -91,7 +91,7 @@ <h2 id="stories-header">Multi-post Stories</h2>
           </div>
         </div>
         <div class="tonic">
-          <div><img src="images/snap1.png" alt="snap" /></div>
+          <div><img src="images/Snapshoot-Portfolio-3.svg" alt="snap" /></div>
```

Comment:
Kindly update the alt text so that it conveys the purpose of the respective image as described above 😃